.github/workflows/test.yaml

```diff
@@ -38,10 +39,68 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
-        if: always()
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Save Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Build an empty test Docker image.
+        run: |
+          docker build --tag empty .
```

Comment:
I didn't realize that building any image whatsoever, even one from scratch creates a lot of images, and the resulting cache size is ~1,373 MB. Let's just pull a small image (maybe an alpine image?) down from GHCR (GitHub Container Registry) instead.

Reply:
[Zipkin is a distributed tracing system](https://zipkin.io/), and they don't look to be in the business of maintaining Docker images, so there's a risk that they might delete that image that may be intended for internal consumption only. [LinuxServer's base images](https://github.com/linuxserver?q=baseimage) look more promising as [LinuxServer maintains a fleet of Docker images](https://www.linuxserver.io/). The smallest of their images that I have found is, unsurprisingly, [baseimage-alpine](https://github.com/linuxserver/docker-baseimage-alpine/pkgs/container/baseimage-alpine) at 21.7 MB. Alpine images are deliberately minimal.